kuiper-game/Services/TimeService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Dawn;
+using Kuiper.Domain;
+
+namespace Kuiper.Services
+{
+    public class TimeService : ITimeService
+    {
+        public static DateTimeOffset RealTimeEpoch { get; private set; }
+        private static readonly DateTimeOffset GameTimeEpoch = new DateTimeOffset(new DateTime(2078, 1, 8));
+        private const int TickAccelerationConstant = 7; //1 real day is 7 game days
+
+        public static void Init(DateTimeOffset realTimeEpoch)
+        {
+            RealTimeEpoch = realTimeEpoch;
+        }
+        
+        [Obsolete("Use the non-static method Now() instead")]
```

Comment:
Since this codebase is not a lib, consider simply removing the method.

Reply:
Yeah, but now - at least you get a warning 😜 This static access pattern is riddled across the codebase (and I was lazy enough not to fix that)

Reply:
so lazy! :P